Anima/registration/anatomical-commands/pyramidal_bm_registration/animaPyramidalBMRegistration.cxx

```diff
@@ -123,19 +123,8 @@ int main(int argc, const char** argv)
     matcher->SetOutputNearestRigidTransformFile(outputNRTransformArg.getValue());
     matcher->SetOutputNearestSimilarityTransformFile(outputNSTransformArg.getValue());
 
-    typedef itk::ImageFileReader<InputImageType> ReaderType;
-
-    ReaderType::Pointer tmpRead = ReaderType::New();
-    tmpRead->SetFileName(fixedArg.getValue());
-    tmpRead->Update();
-
-    matcher->SetReferenceImage(tmpRead->GetOutput());
-
-    tmpRead = ReaderType::New();
-    tmpRead->SetFileName(movingArg.getValue());
-    tmpRead->Update();
-
-    matcher->SetFloatingImage(tmpRead->GetOutput());
+    matcher->SetReferenceImage(anima::readImage <InputImageType> (fixedArg.getValue()));
```

Comment:
It replaces previous lines ?

Reply:
Yep, the magic of RP's utility functions :)